src/main/java/unicash/model/category/UniqueCategoryList.java

```diff
@@ -199,4 +200,38 @@ public static boolean categoriesAreUnique(List<Category> categories) {
         }
         return true;
     }
+
+    /**
+     * Returns all categories as a single, unified string.
+     *
+     * @return all categories as a string
+     */
+    public String joinCategoriesAsString() {
+        StringBuilder categoryBuilder = new StringBuilder();
+
+        internalList.stream().forEach(category -> {
+            categoryBuilder.append(category);
+            categoryBuilder.append(",");
+        });
```

Comment:
Actually, I realise this would leave a trailing `,` which you have to account for, maybe a cleaner way would be

```java
String categories = internalList
    .stream()
    .map(c -> c.toString())
    .collect(Collectors.joining(","));
```

This way, we don't have the issue of a trailing `,` and there's no need to use a `StringBuilder`

Reply:
Yes I was trying to think of ways to prevent that issue as well, thanks for the suggestion!